src/components/Input.js

```diff
@@ -1,20 +1,29 @@
+import { forwardRef } from 'react'
 import styled from 'styled-components/macro'
 import PropTypes from 'prop-types'
 
+const Input = forwardRef((props, ref) => (
+    <InputStyled
+        placeholder={props.placeholder}
+        onChange={props.onChange}
+        value={props.value}
+        required={props.required}
+        ref={ref}
+    />
+))
+
 const InputStyled = styled.input`
     width: 100%;
     position: relative;
     padding-left: 10px;
-    padding-right: 30%;
+    padding-right: 39%;
```

Comment:
You could perhaps use px here, because padding and margin will look better in responsive, this way. You could place your object differently, maybe with flexbox or grid, in your parent component? :)